content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md

```diff
@@ -0,0 +1,225 @@
+while maintaining the original API.
+The [Azure Disk](https://github.com/kubernetes/enhancements/issues/1490)
+and [OpenStack Cinder](https://github.com/kubernetes/enhancements/issues/1489) plugins
+have both been migrated.
+
+### gRPC Probes Graduate to Beta
+
+With Kubernetes 1.24, the [gRPC probes functionality](https://github.com/kubernetes/enhancements/issues/2727) 
+has entered beta and is available by default. You can now [configure startup, liveness, and readiness probes](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) for your gRPC app 
+natively within Kubernetes, without exposing an HTTP endpoint or
+using an extra executable.
+
+### Kubelet Credential Provider Graduates to Beta
+
+Originally released as Alpha in Kubernetes 1.20, the kubelet's support for
+[image credential providers](/docs/tasks/kubelet-credential-provider/kubelet-credential-provider/)
+has now graduated to Beta.
+This allows the kubelet to dynamically retrieve credentials for a container image registry
+using exec plugins, rather than storing credentials on the node's filesystem.
+
+### Contextual Logging in Alpha
+
+Kubernetes 1.24 has introduced [contextual logging](https://github.com/kubernetes/enhancements/issues/3077) 
+that enables the caller of a function to control all aspects of logging (output formatting, verbosity, additional values and names).
+
+### Avoiding Collisions in IP allocation to Services
+
+Kubernetes 1.24 introduced a new opt-in feature that allows you to
+[soft-reserve a range for static IP address assignments](/docs/concepts/services-networking/service/#service-ip-static-sub-range)
+to Services.
+With the manual enablement of this feature, the cluster will prefer automatic assignment from 
+the pool of Service IP addresses thereby reducing the risk of collision. 
+
+A Service `ClusterIP` can be assigned:
+
+* dynamically, which means the cluster will automatically pick a free IP within the configured Service IP range.
+* statically, which means the user will set one IP within the configured Service IP range.
+
+Service `ClusterIP` are unique, hence, trying to create a Service with a `ClusterIP` that has already been allocated will return an error.
+
+## CNI Version-Related Breaking Change
+
+Before you upgrade to Kubernetes 1.24, please verify that you are using/upgrading to a container 
+runtime that has been tested to work correctly with this release.
+
+For example, the following container runtimes are being prepared, or have already been prepared, for Kubernetes:
+
+* containerd v1.6.4 and later, v1.5.11 and later
+* CRI-O 1.24 and later
+
+Service issues exist for pod CNI network setup and tear down in containerd v1.6.0-v1.6.3 in cases where the 
+CNI plugins have not been upgraded and/or the CNI config version is not declared in the CNI config files. 
+These issues are resolved in containerd v1.6.4. 
```

Comment:
Can we liken this statement to reflect the same content as that in the doc?

Reply:
Yep, I can copy it over verbatim.